.gitignore

```diff
@@ -87,3 +87,7 @@ ENV/
 
 # Rope project settings
 .ropeproject
+
+
+# Pycharm settings
+.idea/
```

Comment:
Can we please add our favorite editor's directory to our global git ignore files? I don't think this belongs in a shared repository.